hazelcast-jet-core/src/test/java/com/hazelcast/jet/core/metrics/JobMetrics_MiscTest.java

```diff
@@ -63,6 +63,25 @@ public void setup() {
         TestProcessors.reset(MEMBER_COUNT * parallelism);
     }
 
+    @Test
+    public void when_jobMetricsDisabled_then_emptyMetrics() throws Throwable {
+        DAG dag = new DAG();
+        dag.newVertex("v1", MockP::new);
+        dag.newVertex("v2", (SupplierEx<Processor>) NoOutputSourceP::new);
+        Job job = jet().newJob(dag);
+
+        //when
+        NoOutputSourceP.executionStarted.await();
+        assertJobStatusEventually(job, JobStatus.RUNNING);
+        //then
+        assertTrueAllTheTime(() -> assertEmptyJobMetrics(job), 1);
```

Comment:
1 second is pretty short, collection interval is 5 seconds by default, I don't see it shortened in this test.